### PR TITLE
RF TC_011.010.001 | Verify each image includes a short description of the promotion

### PR DESCRIPTION
**Pull request by elenapozhydaieva**
RF TC_011.010.001
https://trello.com/c/O0iYXhy1

### CODE REVIEW

File: pages/sale_page.py [hunk 1]
```diff
@@ -50,6 +50,14 @@ def is_mens_bargains_text_visible():
 def is_shop_mens_deals_text_visible():
     s(SalePageLocators.SHOP_MENS_DEALS).should(have.text('Shop Men’s Deals'))
 
+def is_the_promotion_text_of_each_image_presented():
+    s(SalePageLocators.BLOCK_PROMO_SALE_20_OFF_TITLE).should(have.text('20% OFF'))
+    s(SalePageLocators.BLOCK_PROMO_SALE_20_OFF_INFO).should(have.text('Every $200-plus purchase!'))
+    s(SalePageLocators.BLOCK_PROMO_SALE_FREE_SHIPPING_TITLE).should(have.text('Spend $50 or more — shipping is free!'))
```
Comment (aqaestuser) [inline pages/sale_page.py:56]: 4. Тестовые данные прокидываем только в тестах

File: tests/test_sale_page.py [hunk 2]
```diff
@@ -81,16 +81,8 @@ def test_011_007_002_clickability_button():
...
-    browser.element(SalePageLocators.BLOCK_PROMO_SALE_20_OFF_INFO).should(have.text('Every $200-plus purchase!'))
-    browser.element(SalePageLocators.BLOCK_PROMO_SALE_FREE_SHIPPING_TITLE).should(
-        have.text('Spend $50 or more — shipping is free!'))
-    browser.element(SalePageLocators.BLOCK_PROMO_SALE_FREE_SHIPPING_INFO).should(have.text('Buy more, save more'))
-    browser.element(SalePageLocators.BLOCK_PROMO_SALE_WOMENS_T_SHIRTS_TITLE).should(
-        have.text('You can\'t have too many tees'))
-    browser.element(SalePageLocators.BLOCK_PROMO_SALE_WOMENS_T_SHIRTS_INFO).should(
-        have.text('4 tees for the price of 3. Right now'))
+    sale_page.visit_sale()
+    sale_page.is_the_promotion_text_of_each_image_presented()
```
Comment (aqaestuser) [inline tests/test_sale_page.py:85]: тут должны быть тестовые данные

**Review by sweetscripts — APPROVED** [review]
Looks good.

**Review by kattypatty — APPROVED** [review]

**Comment by elenapozhydaieva** [comment]
[TC_011.010.001 I Sale > Advertising blocks concerning discount actions and sales > Verify each image includes a short description of the promotion and its benefits](https://trello.com/c/O0iYXhy1/341-tc011010001-i-sale-advertising-blocks-concerning-discount-actions-and-sales-verify-each-image-includes-a-short-description-of-th)

**Pull request merged**